font-src/support/ligation-data.js

```diff
@@ -17,18 +16,12 @@ module.exports = function applyLigationData(data, para, argv) {
 		if (!comp.tag) continue;
 
 		const ligSets = createBuildup(data.simple, comp.buildup);
-		if (comp.isOptOut) {
-			optOutBuildup[comp.tag] = ligSets;
-		} else {
-			optInBuildup[comp.tag] = ligSets;
-		}
-		if (!comp.isOptOut) {
-			hives["ligset-" + gr] = { caltBuildup: ligSets };
-		}
+		defaultBuildup[comp.tag] = ligSets;
+		hives["ligset-" + gr] = { caltBuildup: ligSets };
```

Comment:
We may need to rename these internal hive names, like into `ligset-inherit-<>`, `ligset-enable-<>` and `ligset-disable-<>`.